app/mailers/participant_validation_mailer.rb

```diff
@@ -9,6 +9,8 @@ class ParticipantValidationMailer < ApplicationMailer
   ENGAGE_BETA_MENTOR_TEMPLATE = "99dacb2d-2255-4aa8-9076-46fa6093f1e5"
   INDUCTION_COORDINATOR_NOTIFICATION_TEMPLATE = "d560fb2e-243d-48b1-bf61-c7e111f56858"
   INDUCTION_COORDINATOR_NOTIFICATION_UR_TEMPLATE = "afb54050-7ebd-43af-ad83-7fa6795d1523"
+  # TODO: add real uuid, this is a dummy one
+  INDUCTION_COORDINATOR_VALIDATION_NOTIFICATION_TEMPLATE = "4eb4455d-6b33-4af3-aa40-43b47bfc5501"
```

Comment:
Not sure this is the notification email, is it? This might need renaming

Reply:
I don't know, we are notifying them about the validation changes aren't we? Do you have a suggestion?

Reply:
In beta, we also sent a notification that we've asked their participants to validate. If we're still going to do that, this needs to be called something like "prompt to update". If not, the other templates should probably get a BETA in the name (then be deleted once we know we don't need them). Asked here: https://ukgovernmentdfe.slack.com/archives/C01J1J4FM50/p1630492003125600